Description: Do not run tests
 Especially mysql or, generally, tests requiring a running server.
Forwarded: not-needed
Origin: vendor
Author: Guido De Rosa <guido.derosa@vemarsas.it>
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -45,7 +45,7 @@
 option(SOCI_STATIC "Enable build of static libraries" ON)
 boost_report_value(SOCI_STATIC)
 
-option(SOCI_TESTS "Enable build of collection of SOCI tests" ON)
+option(SOCI_TESTS "Enable build of collection of SOCI tests" OFF)
 boost_report_value(SOCI_TESTS)
 
 # Put the libaries and binaries that get built into directories at the
